src/chi/components/tables/tables.scss

```diff
@@ -134,6 +134,60 @@ th {
     }
   }
 
+  &.-portal {
+    thead {
+      tr {
+        th {
+          background-color: set-color(grey, 25);
+          border-bottom: 0;
+          font-weight: 700;
```

Comment:
@janibolkvadze please remove this property as we have 600 by default in our th and 700 is too heavy